src/apollo-server/index.ts

```diff
@@ -122,6 +123,9 @@ export class OttofellerApolloServerProject extends TypeScriptAppProject {
       },
     })
 
+    // ANCHOR Setup git hooks with Husky
+    addHusky(this, options)
```

Comment:
@quesabe I don’t really like the idea of checking the `hasHusky` option in `addJusky` function. Isn’t it parent caller’s responsibility? What do you think?